examples/react-client/videoroom/src/components/VideoTracks.tsx

```diff
@@ -0,0 +1,30 @@
+import VideoPlayer from "./VideoPlayer";
+import { Track } from "@fishjam-cloud/react-client";
+
+type Props = {
+  id: string;
+  name: string;
+  videoTracks: Track<unknown>[] | undefined;
```

Comment:
**question:** what's the reason behind not using the optional parameter syntax?

Reply:
Anyways - is there a case in which `videoTracks` are going to be `undefined`? I think the SDK should return at least an empty list.

Reply:
There is important difference between `videoTracks: Track<unknown>[] | undefined` and `videoTracks?: Track<unknown>[]`. So they can't be used interchangable.
Anyway, I removed `undefined` as it is indeed not required.